source/test-your-integration/using-integration-for-testing.html.md.erb

```diff
@@ -49,18 +49,18 @@ GOV.UK One Login does not provide specific recommendations about automated testi
 
 
 However, if you choose to do automated testing, you might need to: 
-    
-* embed the integration environment’s basic authentication in a URL
+
+* enter the integration environment’s basic authentication in a URL
 * generate a one-time code using a scripting language
 
 
 ### Embed the integration environment’s basic authentication in a URL
 
 
 The integration environment web user journey will present a basic authentication challenge to protect the environment from accidental use. This will prompt you to log in and might disrupt your automated tests. 
-    
-    
-You can use a URL with the integration environment’s basic authentication embedded within it when you make a request to the `/authorize` endpoint: `https://username:password@signin.integration.account.gov.uk/authorize`.
+
+
+You can call the URL: https://username:password@signin.integration.account.gov.uk before you make a request to the `/authorize` endpoint. This will pass the required credentials automatically bypassing the login prompt.    
```

Comment:
```suggestion
You can visit the URL: https://username:password@signin.integration.account.gov.uk before you make a request to the `/authorize` endpoint. This will pass the required credentials automatically bypassing the login prompt.    
```